manifest/autoversion/testdata/github.expected.hcl

```diff
@@ -18,14 +18,6 @@ version "3.2.137" "3.2.140" "3.2.150" {
   }
 }
 
-version "5.1.233" "5.1.234" {
```

Comment:
Note to reviewers:
Autoversioning now tries to download the latest version into cache. Non existent versions like this one cause the test to fail. So I have removed them. That should be fine in my opinion but I am open to thoughts.